spec/features/user_edits_snippet_spec.rb

```diff
@@ -18,8 +18,8 @@
   end
 
   scenario "unsuccessfully" do
```

Comment:
Should this specify the conditions that lead to it being unsuccessful? Ex., "When the artist name is blank" / "user cannot edit the snippet" or something like that?